src/libs/ReportUtils.js

```diff
@@ -4264,45 +4264,14 @@ function getIOUReportActionDisplayMessage(reportAction) {
 }
 
 /**
- * Return room channel log display message
+ * Return room channel log message
  *
  * @param {Object} reportAction
  * @returns {String}
  */
-function getChannelLogMemberMessage(reportAction) {
-    const verb =
-        reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ROOMCHANGELOG.INVITE_TO_ROOM || reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG.INVITE_TO_ROOM
-            ? 'invited'
-            : 'removed';
-
-    const mentions = _.map(reportAction.originalMessage.targetAccountIDs, (accountID) => {
-        const personalDetail = lodashGet(allPersonalDetails, accountID);
-        const displayNameOrLogin =
-            LocalePhoneNumber.formatPhoneNumber(lodashGet(personalDetail, 'login', '')) || lodashGet(personalDetail, 'displayName', '') || Localize.translateLocal('common.hidden');
-        return `@${displayNameOrLogin}`;
-    });
-
-    const lastMention = mentions.pop();
-    let message = '';
-
-    if (mentions.length === 0) {
-        message = `${verb} ${lastMention}`;
-    } else if (mentions.length === 1) {
-        message = `${verb} ${mentions[0]} and ${lastMention}`;
-    } else {
-        message = `${verb} ${mentions.join(', ')}, and ${lastMention}`;
-    }
-
-    const roomName = lodashGet(reportAction, 'originalMessage.roomName', '');
-    if (roomName) {
-        const preposition =
-            reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ROOMCHANGELOG.INVITE_TO_ROOM || reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG.INVITE_TO_ROOM
-                ? ' to'
-                : ' from';
-        message += `${preposition} ${roomName}`;
-    }
-
-    return message;
+function getChannelLogMemberMessagePlainText(reportAction) {
```

Comment:
Would you move this to `ReportActionsUtils.ts`? It's a better place. We're basically rewriting this function, so we can as well move it.